front/components/app/DatasetView.tsx

```diff
@@ -325,9 +325,9 @@ export default function DatasetView({
     setDatasetData(data);
     setDatasetTypes([]);
     onUpdate(datasetInitializing, datasetTypesValidation(), {
-      name: datasetName,
+      name: datasetName.slice(0, 255),
```

Comment:
Can we have a const like DATA_SOURCE_NAME_MAX_LENGTH=255 somewhere to make sure it's always in sync?